feature/repair/src/main/res/values/strings.xml

```diff
@@ -11,7 +11,7 @@
     <string name="label_pet_in_unit">Pet in Unit</string>
     <string name="hint_pet_in_unit">Category</string>
     <string name="text_service">I would like to request a service for…</string>
-    <string name="label_category">Select Category…</string>
```

Comment:
Why this changed was required? In the UI, it says `Select Category…`

Reply:
Because I changed for string -->  "Category (scroll for more options)"
This is more accurate since the user will not know that they can scroll the options. It is not related to the fix LP166 but i just made the small change here, to not create an entire branch for it. So, in my opinion, this should change :)

Reply:
Mmm  I think it clearer the text `Select category...`. Can you change it to the way it was before please?

Reply:
The thing is that the field already has a "label" that says -> "I would like to request a service for .."
Then, the one that you are mentioning, is the "hint". So I really believe we should leave the (select category), because when opening the options available, it is really hard to know that you can actually scroll. The options do not fit the length of the dropboxfield (The image does show the previous, but just included so you can see the label and hint location.

![image](https://user-images.githubusercontent.com/25538206/235747737-8ae27cc8-da72-4535-a35f-1cd569b58f8c.png)

Reply:
It is not hard to know that you can do scroll because all the select inputs do this thing. With the arrow the user could see all the options and do scroll to see more, so I think with the text select category is enough

Reply:
This is an example: to see all the categories you need to do a scroll but in the hint text you don't see `(scroll for more options)`

![WhatsApp Image 2023-05-02 at 2 37 25 PM (1)](https://user-images.githubusercontent.com/105077839/235755810-6a96d4c9-57dc-4053-93df-961ee0a54986.jpeg)
![WhatsApp Image 2023-05-02 at 2 37 25 PM](https://user-images.githubusercontent.com/105077839/235755831-8ba63f91-9647-4e5b-952e-ff0981d136e0.jpeg)

Reply:
I think you are not getting the point or I am not explainin enough. For example, the Pet in Unit field, when you open it. It will show all the options. The scroll does not appear here because options display fully.

![image2](https://user-images.githubusercontent.com/25538206/235755376-7f794a59-9605-4291-9adf-7daf8cf51ca6.png)

Then, for the Category field, because it has plenty of fields, it will not show all the options on the entire square. Therefore, the user has to scroll to visualize all the options. I think that if we do not add the explicit "scroll" in text, there could be times where user is not aware of all the available options.
![Screenshot 2023-05-02 143659](https://user-images.githubusercontent.com/25538206/235755488-ec0a3326-7198-40fd-a402-4bc33f4b5fed.png)
![Screenshot 2023-05-02 143659](https://user-images.githubusercontent.com/25538206/235756467-ed161aab-6ffc-487d-9807-1ea1bcbaa336.png)

Reply:
Actually I get your point, but in android is not common see the hint `scroll for more options`. If you use other apps (the one I show you is Discover app) the input select at the beginning you cannot see the scroll until the user does... So is going to be the same in this example, the select that I show you in the screenshots do the same that you are pointing, and as user, we can do scroll and implicitly the user does scroll without know  it... so I still think that is not necessary being explicit because everyone knows how could function the select. 
So I still think that it is enough with `Select category`

Reply:
As you can see, It has the same behavior that you mention and it does not have the text scroll to see more options

https://user-images.githubusercontent.com/105077839/235758623-1ec97cdf-ad1f-4606-83d3-68ff5447afde.mp4

Reply:
In fact, there is an error with the input select: the option's list should be the same width that the input

Reply:
Ok, I saw the app. I still think that it should be left with "something" that the user is able to identify that is scrollable (During a development of an app and usability test that i was part of most users did not know that it was scrollable). But if you say that, I will believe it. 
Talking about the component error, that is an error of the actual component, not of the view Repair itself. Maybe we should ask scrum master to assign the fix to the correspondent developer.

I will change the text string.